frontend/src/components/maps/leaflet/Map.tsx

```diff
@@ -289,86 +301,112 @@ const Map: React.FC<MapProps> = ({
+                    onzoomend={e => setZoom(e.sourceTarget.getZoom())}
+                    onmoveend={handleBounds}
+                  >
+                    {activeBasemap && (
+                      <TileLayer
+                        attribution={activeBasemap.attribution}
+                        url={activeBasemap.url}
+                        zIndex={0}
                       />
-                    </Popup>
-                  )}
-                  <LegendControl />
-                  <ZoomOutButton map={mapRef} bounds={defaultBounds} />
-                  <LayersControl />
-                  <InventoryLayer
-                    zoom={zoom}
-                    bounds={bounds}
-                    onMarkerClick={onSingleMarkerClick}
-                    selected={selectedProperty}
-                    filter={geoFilter}
-                  ></InventoryLayer>
-                </ReactLeafletMap>
-              </Col>
-            </Row>
-          </Container>
-        );
-      }}
-    </ReactResizeDetector>
+                    )}
+                    {!!layerPopup && (
+                      <Popup
+                        position={layerPopup.latlng}
+                        offset={[0, -25]}
+                        onClose={() => setLayerPopup(undefined)}
+                        closeButton={interactive}
+                        autoPan={false}
+                      >
+                        <LayerPopupTitle>{layerPopup.title}</LayerPopupTitle>
+                        <LayerPopupContent
+                          data={layerPopup.data as any}
+                          config={layerPopup.config as any}
+                          center={layerPopup.center}
+                          onAddToParcel={(e: MouseEvent, data: { [key: string]: string }) => {
+                            dispatch(saveParcelLayerData({ e, data }));
+                          }}
+                          bounds={layerPopup.bounds}
+                        />
+                      </Popup>
+                    )}
+                    <LegendControl />
+                    <ZoomOutButton map={mapRef} bounds={defaultBounds} />
+                    <LayersControl
+                      open={layersOpen}
+                      setOpen={() => {
+                        setLayersOpen(!layersOpen);
+                        setInfoOpen(false);
+                      }}
+                    />
+                    <InfoSlideOut
+                      open={infoOpen}
+                      setOpen={() => {
+                        setInfoOpen(!infoOpen);
+                        setLayersOpen(false);
+                      }}
+                    />
+                    <InventoryLayer
+                      zoom={zoom}
+                      bounds={bounds}
+                      onMarkerClick={() => {
+                        if (!infoOpen) {
+                          setLayersOpen(false);
+                          setInfoOpen(true);
```

Comment:
don't close the slide out when a marker is clicked again: if one marker is clicked then a different one is clicked right after, the slide out stays open